src/logic/addressBook/utils/index.ts

```diff
@@ -1,5 +1,7 @@
+import { List } from 'immutable'
 import { loadFromStorage, saveToStorage } from 'src/utils/storage'
 import { AddressBookEntryProps } from './../model/addressBook'
+import { SafeOwner } from '../../../routes/safe/store/models/safe'
```

Comment:
can we use absolute paths instead of relative ones?

```suggestion
import { SafeOwner } from 'src/routes/safe/store/models/safe'
```